api/Navigator.json

```diff
@@ -55,53 +55,112 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Navigator/authentication",
           "support": {
             "chrome": {
-              "version_added": "40"
+              "version_added": "67",
+              "flags": [
+                {
+                  "name": "WebAuth",
+                  "type": "preference"
+                }
+              ]
```

Comment:
I won't know for a few days yet whether this will actually be in Chrome 67. It will also be on by default, not behind a flag.

(The name of the preference is enable-web-authentication-api. The IDL name is not what shows up in chrome://flags.)

Reply:
That was merged as part of #1742.